homeassistant/components/google_assistant/helpers.py

```diff
@@ -426,6 +426,7 @@ async def sync_serialize(self, agent_user_id):
                 "webhookId": self.config.local_sdk_webhook_id,
                 "httpPort": self.hass.http.server_port,
                 "httpSSL": self.hass.config.api.use_ssl,
+                "uuid": await self.hass.helpers.instance_id.async_get(),
```

Comment:
I wonder if this is something that we should pass in. That way we won't do an `await` inside a for-loop (we always serialize inside a loop).

Oh see we don't do that either for all the registries. We should really have some InstanceContext object that we pass in that conatains all 4 but we can do that in a future PR.

Reply:
Also same for the `get_url`. We could compute that just once.

Reply:
Yes, that was my first idea, but a thought I would keep the same style as the others